src/components/header.js

```diff
@@ -16,7 +16,7 @@ const Header = () => {
         site_title_image {
           childImageSharp {
             fluid {
-              ...GatsbyImageSharpFluid
+              ...GatsbyImageSharpFluid_withWebp_tracedSVG
```

Comment:
TracedSVG:
https://using-gatsby-image.gatsbyjs.org/traced-svg/
pretty cool!
and if you're curious what WebP is: https://developers.google.com/speed/webp